test/controllers/thinkers_controller_test.rb

```diff
@@ -1,14 +1,49 @@
 require 'test_helper'
 
 class ThinkersControllerTest < ActionController::TestCase
+  def setup
+    @thinker = Thinker.create( name: "Example Thinker", email: "thinker@example.com")
+  end
+
   test "should get new" do
     get :new
     assert_response :success
   end
 
-  #test "should get destroy" do
-   # get :destroy
-   # assert_response :success
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:thinkers)
+  end
+
+  test "should create thinker" do
+	  assert_difference('Thinker.count') do
+	  post :create, thinker: {name: 'Some name', email: 'email@example.com'}
+	  end
+ 
+  	assert_redirected_to thinker_path(assigns(:thinker))
```

Comment:
We could improve test coverage here by verifying the values for the assigned thinker (name, email) match the parameters given.